src/pages/home/home.ts

```diff
@@ -832,20 +833,20 @@ export class HomePage {
     this.externalLinkProvider.open(url);
   }
 
-  public enableBitPayIdPairing() {
+  public toggleTestnet() {
     this.tapped++;
-
     if (this.tapped >= 10) {
-      this.persistenceProvider.getBitpayIdPairingFlag().then(res => {
-        res === 'enabled'
-          ? this.persistenceProvider.removeBitpayIdPairingFlag()
-          : this.persistenceProvider.setBitpayIdPairingFlag('enabled');
-
-        alert(
-          `BitPay ID pairing feature ${res === 'enabled' ? res : 'disabled'}`
-        );
-        this.tapped = 0;
-      });
+      this.persistenceProvider
+        .getNetwork()
+        .then((currentNetwork: Network | undefined) => {
+          const newNetwork =
+            !currentNetwork || currentNetwork === Network.livenet
+              ? Network.testnet
+              : Network.livenet;
+          this.persistenceProvider.setNetwork(newNetwork);
+          alert(`Network changed to ${newNetwork}. Please restart the app.`);
```

Comment:
I'd avoid to use `alert` function. Maybe you could restart the app automatically. Take a look: https://github.com/bitpay/wallet/blob/master/src/pages/settings/wallet-settings/wallet-settings-advanced/wallet-service-url/wallet-service-url.ts#L119

Reply:
Ok i added the automatic reload but changed the alert to use an info sheet. Just to avoid confusion for testers, I wanted to have an indication of what network they are using.